analysis.ml

```diff
@@ -320,3 +320,28 @@ let aliased ({formals; instrs} : analysis_input) : pc -> VarSet.t =
   in
   let mut_formals = List.fold_left ref_param [] (ModedVarSet.elements formals) in
   fun _ -> VarSet.of_list mut_formals
+
+module Expression = struct
+  type t = expression
+  let compare (x : expression) (y : expression) =
+    Pervasives.compare
+      (Disasm.disassemble_exp_s x)
+      (Disasm.disassemble_exp_s y)
```

Comment:
Doesn't `Pervasives.compare x y` work just as well? Are there expressions that have different representations but that you want to consider equal?